### PR TITLE
Add tests for about page i18n

### PR DESCRIPTION
**Pull request by sonalkr132**
I am not sure if this is the right way to do it.. :neutral_face:

### CODE REVIEW

**Comment by dwradcliffe** [comment]
For the translations this is basically what I was going to do. We can let others send PRs to fix them for each language. Not great but I don't have a better idea and I want to un-break these pages.

File: test/integration/about_page_test.rb [hunk 1]
```diff
@@ -0,0 +1,23 @@
+require 'test_helper'
+
+class AboutPageTest < ActionDispatch::IntegrationTest
+  def assert_about_page_i18n(local)
+    get page_url("about"), locale: local
+
+    assert_response :success
+    assert_select(".t-body a") do |elements|
+      assert_equal 13, elements.count
+    end
```
Comment (dwradcliffe) [inline test/integration/about_page_test.rb]: I don't think we need to check this closely, just as long as we get a 200 response and this fails if we miss something (like I did yesterday).

File: test/integration/about_page_test.rb [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+  test "about page i18n for all supported languages" do
+    assert_about_page_i18n 'en'
+    assert_about_page_i18n 'es'
+    assert_about_page_i18n 'de'
+    assert_about_page_i18n 'nl'
+    assert_about_page_i18n 'fr'
+    assert_about_page_i18n 'pt-BR'
+    assert_about_page_i18n 'zh-CH'
+    assert_about_page_i18n 'zh-TW'
```
Comment (dwradcliffe) [inline test/integration/about_page_test.rb]: You can get the list from `I18n.available_locales` so we don't need to hardcode the available locales here.

**Comment by sonalkr132** [comment]
updated!

**Pull request merged**